corehq/apps/hqmedia/views.py

```diff
@@ -659,22 +663,38 @@ def get(self, request, *args, **kwargs):
         return response
 
 
-def iter_index_files(app, build_profile_id=None):
+def iter_index_files(app, build_profile_id=None, download_targeted_version=False):
     from corehq.apps.app_manager.views.download import download_index_files
     from dimagi.utils.logging import notify_exception
-    skip_files = ('profile.xml', 'profile.ccpr', 'media_profile.xml')
+    skip_files = (
```

Comment:
Would it make sense to store `commcare` and `commcare_lts` in a constant somewhere so you could generate this automatically?

Reply:
https://github.com/dimagi/commcare-hq/pull/20235